### DISCUSSION TITLE
Broken Track Records

### DISCUSSION THREAD

**Discussion by pattoco** [osmandapp/OsmAnd#14232] [Q&A]
Hello, sorry for my bad english

Me and a friend use Osmand to record tours.

Everything works for me and you can see that the recording follows the streets.

![1](https://user-images.githubusercontent.com/105226755/167440080-7a64d34c-d93d-4f50-b4fb-f6a02e98b868.jpg)

But my friend has something wrong with the recording and looks like he's flying over the map.

![kaputt](https://user-images.githubusercontent.com/105226755/167440389-e580e59b-81ac-4f4c-85ea-587961c510b7.jpg)

We both have the same smartphones, version and settings.

Does somebody has any idea?

**Comment by vshcherb** [comment]
It could be personal (AI ?) of power management - https://dontkillmyapp.com/
